sox.features.js

```diff
@@ -496,22 +496,15 @@
     shareLinksMarkdown: function() {
       // Description: For changing the 'share' button link to the format [name](link)
 
-      const questionTarget = document.getElementById('question');
-      const answersTargets = document.getElementsByClassName('answer');
-      sox.helpers.observe([questionTarget, ...answersTargets], '.share-tip', () => {
-        const input = document.querySelector('.share-tip input');
-        if (!input) return;
-
-        const title = document.querySelector('meta[name="twitter:title"]').getAttribute('content').replace(/\[(.*?)\]/g, '[$1]'); // https://github.com/soscripted/sox/issues/226, https://github.com/soscripted/sox/issues/292
-        const link = input.value;
-
-        // Do nothing if the function's already done its thing
-        if (link.indexOf(title) !== -1) return;
-
-        input.value = `[${title}](${link})`;
-        input.select();
-        document.execCommand('copy'); // https://github.com/soscripted/sox/issues/177
-      });
+      const title = $('.fs-headline1.fl1').contents().html().replace(/\[(on\shold|duplicate)\]/g, '($1)'); // https://github.com/soscripted/sox/issues/226, https://github.com/soscripted/sox/issues/292
```

Comment:
I think the old method of selecting the `twitter:title` meta tag's text is a bit better because it means we don't need to do the replacing ourselves which we don't really need to do if there's a place we can get it in the form we want already!

Reply:
Selecting the `twitter:title` meta tag's text, though, won't have question's state (on hold/duplicate)! However, in my case, it is included and it is replaced with `(on hold/duplicate)`.

Reply:
Oh, you're right - sorry, my bad! 

Could we change `$('.fs-headline1.fl1').contents().html()` to `$('.fs-headline1.fl1').text()` to simplify it a bit? I think they should both do the same thing in this case. (also I'm not sure if we want the HTML here or just the text?)

Reply:
OK, I have simplified that as you said ;)